xyzservices/lib.py

```diff
@@ -252,6 +252,49 @@ def _filter_bunch(bunch, keyword, name, requires_token, function):
             function=function,
         )
 
+    def query_name(self, name: str) -> TileProvider:
+        """Return :class:`TileProvider` based on the name query
+
+        Returns a matching :class:`TileProvider` from the :class:`Bunch` if the ``name``
+        can be parsed into the string matchinch provider's name. See examples for
```

Comment:
```suggestion
        can be parsed into the string matching the provider's name. See examples for
```

(although the sentence is maybe still not fully correct)

Reply:
If you have a suggestion how to make it correct I am all ears. Sounds okay to me now (not that it means anything when it comes to English).